openmeetings-web/src/main/java/org/apache/openmeetings/web/common/tree/FileTreePanel.java

```diff
@@ -192,7 +191,7 @@ public FileTreePanel(String id, Long roomId, NameDialog addFolder) {
 		final OmTreeProvider tp = new OmTreeProvider(roomId);
 		select(tp.getRoot(), null, false, false);
 		form.add(tree = new FileItemTree("tree", this, tp));
-		form.add(download.setVisible(false).setOutputMarkupPlaceholderTag(true));
+		form.add(download.setEnabled(false).setOutputMarkupPlaceholderTag(true));
```

Comment:
I have changed `enabled` with `visible` since this block is confusing for items without download option
I would like to get third opinion here @dkandrov WDYT?

Reply:
Hiding vs disabling:
https://ux.stackexchange.com/questions/107513/what-is-better-for-ux-hiding-or-disabling-irrelevant-buttons
https://www.theusabilitypeople.com/thought_leadership/disable-hide-or-grey-out


> 1) When there is no way that an end user will EVER have access to a feature or function, do not show it to them! (This is mostly for system admin features/functions that most end users will not need: e.g. Add/Remove users, etc.)

That sums it up quite well for me. I know we also changed exactly this button and section like a 100 times before.

But there is a qualified yes and no to hiding vs disabling and I think above quote sums it up pretty well.

Reply:
I'm OK with it deing disabled
The only concern I have: previously we had `Disabled` room menu items
And I had millions of requests: to remove those

Shall we ask at user@ list?

Reply:
sure I can write it up later and send there.

Reply:
which room menu items are you referring to ?
That might be a bit of a different problem I think. That is around permissions. Not contextual. Above button simply depends on if you click on a document or not.
But _EVERY_ user can see the button. Its not a permission thing. Which I think would be a slightly different use-case again. Cause if you remove permissions form a user to do sth, I would argue probably you should also remove the UI elements, cause that is indeed confusing to disable them, cause the user thinks he just need to click the right icon to enable it.

Reply:
I mean this Room menu

![Screenshot from 2020-04-18 09-39-10](https://user-images.githubusercontent.com/3870591/79626214-94191b80-8158-11ea-857d-eae697d4c146.png)

Previously it was looking like this: (https://om.alteametasoft.com/openmeetings/#room/5)
![Screenshot from 2020-04-18 09-40-31](https://user-images.githubusercontent.com/3870591/79626240-c7f44100-8158-11ea-9f6b-9e82ce8cfe03.png)

NOTE disabled items